__tests__/app.test.js

```diff
@@ -37,3 +37,21 @@ describe('GET /api/topics', ()=>{
         
        })
 })
+
+describe('GET /api', () => {
+    test("200: should return a key value pairs of endpoint objects", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body})=>{
+          const { api } = body
+          for(let key in api){
+            expect(api[key]).toHaveProperty("description", expect.any(String));
+            expect(api[key]).toHaveProperty("queries", expect.any(Array));
+            expect(api[key]).toHaveProperty("format", expect.any(String));
+            expect(api[key]).toHaveProperty("exampleResponse", expect.any(Object));
+          }
+        })
```

Comment:
This test is fine, but we can simplify if desired, by just requiring in the JSON file and expecting it to be sent as the response. I’ll leave it up to you if you want to do this! 🙂